### PR TITLE
Add veusz plotting package

### PR DESCRIPTION
**Pull request by jeremysanders**

### CODE REVIEW

**Comment by jeremysanders** [comment]
Hi - if someone has some insight into why the Windows build is failing, that would be great. It looks like it's having problems finding the Qt header files, though the include directories in the command line look like they should be reasonable.

**Comment by jeremysanders** [comment]
Release now includes a patch which is applied upstream. This uses qmake to find the Qt location, rather than PyQt. See this bug: https://github.com/ContinuumIO/anaconda-issues/issues/9691

**Comment by jeremysanders** [comment]
@conda-forge/staged-recipes This is ready for review now.

I needed the setuptools dependency, as it wouldn't build without that.

**Comment by jeremysanders** [comment]
gentle bump @conda-forge/staged-recipes

File: recipes/veusz/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,69 @@
...
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: {{ sha256 }}
+
+build:
+  number: 0
+  script: python -m pip install --no-deps --ignore-installed .
```
Comment (marcelotrevisani) [inline recipes/veusz/meta.yaml]: Please, change that to:
```
script: "{{ PYTHON }} -m pip install . --no-deps --ignore-installed --no-cache-dir -vvv"
```

File: recipes/veusz/meta.yaml [hunk 2]
```diff
@@ -0,0 +1,69 @@
...
+    - pip
+    - setuptools
+    - qt
+    - sip
+    - pyqt
+    - numpy
+  run:
+    - python
+    - pip
+    - setuptools
```
Comment (marcelotrevisani) [inline recipes/veusz/meta.yaml]: Do you need ``pip`` or ``setuptools`` at runtime?

**Comment by jeremysanders** [comment]
It fails without those in the run time dependencies, giving:

```
+ veusz -platform minimal --help
Traceback (most recent call last):
  File "/home/conda/feedstock_root/build_artifacts/veusz_1536659727561/_test_env_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_pl/bin/veusz", line 6, in <module>
    from pkg_resources import load_entry_point
ModuleNotFoundError: No module named 'pkg_resources'
```

I'll put them back in with a comment

File: recipes/veusz/meta.yaml [hunk 3]
```diff
@@ -0,0 +1,70 @@
...
+    - qt
+    - sip
+    - pyqt
+    - numpy
+  run:
+    - python
+    - qt
+    - pyqt
+    - {{ pin_compatible('numpy') }}
+    # pip/setuptools required, otherwise cannot find pkg_resources module
```
Comment (marcelotrevisani) [inline recipes/veusz/meta.yaml]: Try to remove ``pip`` and also ``setuptools`` and add ``pkg_config``  please

**Comment by jeremysanders** [comment]
It looks like pkg_config doesn't exist separately to setuptools

**Comment by marcelotrevisani** [comment]
I saw that wrong, sorry
You will need ``setuptools`` but not ``pip`` in run requirements

After the CI pass I will merge that, thanks to put this in ``conda-forge``!  :)

**Pull request merged**